Library/Homebrew/rubocops/lines.rb

```diff
@@ -84,6 +84,28 @@ def audit_formula(_formula_nodes)
         end
       end
 
+      # Ensure every `skip_clean` is documented with a comment.
+      class SkipCleanCommented < FormulaCop
+        sig { override.params(formula_nodes: FormulaNodes).void }
+        def audit_formula(formula_nodes)
+          return if formula_tap != "homebrew-core"
+          return if (body_node = formula_nodes.body_node).nil?
+
+          # Some formulae call `skip_clean` multiple times, only require a
+          # comment on the first
+          method = find_every_method_call_by_name(body_node, :skip_clean).first
+          return if method.nil?
+
+          method_comment = processed_source.comments.find do |comment|
+            (method.loc.line - comment.loc.line).abs <= 1
+          end
+          return unless method_comment.nil?
```

Comment:
nit, `unless nil` confuses me :-)

```suggestion
          return if method_comment.present?
```

Reply:
Or

```suggestion
          return if !method_comment.nil?
```

Reply:
I prefer the prior or the existing version to the second.